src/numeric/grids.jl

```diff
@@ -5,19 +5,48 @@ function mlinspace(min, max, n)
     return QE.gridmake(nodes...)
 end
 
-type CartesianGrid <: Grid
```

Comment:
This comment applies to the function `mlinspace` above (it wasn't changed in this PR, so I can't comment on it...)

You can define `nodes = map(linspace, min, max, n)`, which I prefer to the comprehension and splatting from a style perspective